tests/test_api.py

```diff
@@ -186,9 +188,12 @@ def test_change_password(self, fixture_aws_mock_api):
             "JciHitachi.aws_connection.ChangePassword.get_data"
         ) as mock_get_data_1, patch(
             "JciHitachi.connection.UpdateUserCredential.get_data"
-        ) as mock_get_data_2:
+        ) as mock_get_data_2, patch("httpx.post") as mock_post:
             mock_get_data_1.return_value = ("OK", "")
             mock_get_data_2.return_value = ("OK", "")
+            mock_post.return_value = httpx.Response(
+                200, text=json.dumps({"status": {"code": "OK", "message": ""}})
+            )
```

Comment:
I don't think this tackles the root cause - it circumvents the error from occurring in the test indeed but the issue with the real `httpx.post` still exists.